app/controllers/entities_controller.rb

```diff
@@ -0,0 +1,41 @@
+class EntitiesController < ApplicationController
+  # def index
+  #   @group = Group.find(params[:group_id])
+  #   @relations = @group.relations
+  # end
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.